### DISCUSSION TITLE
机器人发送的部分图片PCQQ无法显示

### DISCUSSION THREAD

**Discussion by SinCerely023** [GraiaProject/Application#112] [Q&A] [answered]
机器人发送的一部分图片在手机QQ上是正常的，但是在PC端无法显示。
使用fromLocalFile()和fromUnsafeBytes()都会出现这种问题。
初步判断是uploadImage之后，文件后缀变成了.mirai，PCQQ无法出预览图。
但是为什么一部分图片会出现这种问题呢

**Comment by karakoo** [comment, marked as answer]
这应该是mirai-core对图片消息的封装问题，与graia-project无关
